### ISSUE TITLE
Question : Batch normalization

### ISSUE THREAD

**Issue by kaushalshetty** [keras-team/keras#7896]
Should the normalization we do before inputting it to a neural network depend upon the activation function used? Does batch normalization learn what type of distribution is to be fed to any layer depending on the activation function used ?

Thank You
